src/fontra/client/core/glyph-svg.js

```diff
@@ -31,7 +31,12 @@ export class SVGPath2D {
     this.items.push("L" + this._format(x, y));
   }
 
+  // compatibility method for Path2D
```

Comment:
Why this change? I don't think glyph-svg is relevant for this PR.

Reply:
Done.

Reply:
Had to undo this, because otherwise checks fail:
<img width="922" alt="Screenshot 2024-03-21 at 15 20 24" src="https://github.com/googlefonts/fontra/assets/21055547/16d8902a-c429-4ca5-bba2-34af360b7405">

Reply:
Compare what it looks like in the main branch. This function should not be touched in this PR.

Reply:
Sorry, my bad. done.